browserbase/README.md

```diff
@@ -1,110 +1,48 @@
-# Browserbase MCP Server
+# Playwright Browserbase MCP Server
```

Comment:
I am a bit confused here - didn't Paul say yesterday we want to combine Stagehand and Browserbase MCPs into one?

Reply:
Yes, we can ignore this readme for now.

Ideally getting the foundations of playwright mcp, and enhancing this current mcp with browserbase / stagehand abilities (flexible web automations, recordings, etc.)